google-cloud-logging/lib/google/cloud/logging/rails.rb

```diff
@@ -30,8 +30,10 @@ module Logging
       # {Google::Cloud::Logging::Logger} instance to be used by the Rails
       # application.
       #
-      # The Middleware is only added when certain conditions are met. See
-      # {use_logging?} for details.
+      # The Middleware is only loaded when certain conditions are met. These
```

Comment:
@dazuma What do you think of these changes?

Reply:
The `use_logging` configuration values actually override `Rails.env.production?`. That is, `use_logging` set to `false` will disable the middleware regardless of `Rails.env`. So maybe better:

```
The middleware is loaded only when certain conditions are met. These conditions are when the configuration `Google::Cloud.configure.use_logging` (also available as `Rails.application.config.google_cloud.use_logging` for a Rails application) is set to `true`, or, if the configuration is left unset but `Rails.env.production?` is `true`.
```

Reply:
That is more clear. Updated. PTAL.